apps/airdrop/components/claim/ClaimTokenScreen.tsx

```diff
@@ -53,52 +53,54 @@ export function ClaimTokenScreen() {
     }
   });
 
-  const { executeAsync } = useAction(getAirdropTokenStatusAction, {
-    onError: (response) => {
-      log.error('Error checking airdrop status', { address, chainId, error: response.error });
-      toast.error(response.error?.serverError?.message || 'Error retrieving airdrop claim status');
-    }
-  });
-
   const { switchChainAsync } = useSwitchChain();
   const { refreshBalance } = useDevTokenBalance({ address });
 
   // Move the check logic to a separate function
   const checkAirdropStatus = async (walletAddress: string) => {
     setIsGettingAirdropTokenStatus(true);
     try {
-      const result = await executeAsync({ address: walletAddress });
-      const data = result?.data;
-
-      if (!data) {
+      const result = (await fetch(`/api/airdrop?address=${walletAddress}`).then((res) => res.json())) as {
```

Comment:
needed to convert it to a api route otherwise the action was not working on farcaster frames